lib/run/export-file.js

```diff
@@ -77,9 +82,25 @@ module.exports = function (options, done) {
                     `error creating path for file "${path.unparsed}" for ${options.name || 'unknown-source'}`));
             }
 
-            fs.writeFile(path.unparsed, content, function (err) {
-                done(_.set(err, 'help',
-                    `error writing file "${path.unparsed}" for ${options.name || 'unknown-source'}`), path);
+            fs.stat(path.unparsed, function (error, stat) {
+                var target;
+
+                // handle cases where the specified export path is a pre-existing directory
+                if (stat && stat.isDirectory()) {
```

Comment:
If stat is a file?

Reply:
Then it gets overwritten, which has been the behaviour since exports were introduced.